### PR TITLE
fix(prover_cli): Fix the issues with `home` path

### PR DESCRIPTION
**Pull request by ColoCarletti**
In this PR: [Pull Request #2022](https://github.com/matter-labs/zksync-era/pull/2022/files), the logic was changed to stop using the $ZKSYNC_HOME environment variable to construct paths relative to the root of zksync-era. 
But the prover is a separate workspace, so it fails to create the path to the contract with the functions in the main workspace..

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [x] Code has been formatted via `zk fmt` and `zk lint`.
- [x] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

**Review by Oppen — APPROVED** [review]

File: prover/prover_cli/src/main.rs [hunk 1]
```diff
@@ -5,15 +5,13 @@ async fn main() {
     tracing_subscriber::fmt()
         .with_max_level(tracing::Level::ERROR)
         .init();
-
```
Comment (ilitteri) [inline prover/prover_cli/src/main.rs:8]: Ditto

File: prover/prover_cli/src/main.rs [hunk 2]
```diff
@@ -5,15 +5,13 @@ async fn main() {
...
     config::get_envfile()
         .and_then(config::load_envfile)
         .inspect_err(|err| {
             tracing::error!("{err:?}");
             std::process::exit(1);
         })
         .unwrap();
-
```
Comment (ilitteri) [inline prover/prover_cli/src/main.rs:16]: Ditto.

**Review by ilitteri — COMMENTED** [review]
Everything looks good, left some questions

**Review by ilitteri — APPROVED** [review]
LGTM

**Pull request merged**